### ISSUE TITLE
Uncaught SyntaxError: Unexpected token < in aos.js1

### ISSUE THREAD

**Issue by sikemausa** [michalsnik/aos#115]
I'm using create-react-app and am getting the above error in the console - fresh install and everything. Any ideas?

**Comment by syvzies** [comment]
@sikemausa - I got this same error.  I am using the MEAN stack, but I'm wondering if the problems have the same cause.

In my app.js file, I had:
`app.use(express.static(__dirname + '/public'));`
so that anything in the public folder would be served by express's static function.  I was routing all other requests to my main index.html page since I was building a single page app.

The problem was I had installed the npm package and then was trying to reference that package in the node_modules folder.  Because it wasn't in the public folder, the server was taking my request and responding with the index.html file.  The browser would be expecting js and instead would receive an html file and immediately error.

I'm not familiar with create-react-app, but I would make sure that your server is correctly set up with your folder structure so that you are sending back the correct files.  It's possible a similar thing is happening here and you are trying to request AOS.js and instead are getting your index.html file.
